test/e2e/kubectl/kubectl.go

```diff
@@ -853,6 +853,35 @@ metadata:
 		})
 	})
 
+	ginkgo.Describe("Kubectl diff", func() {
+		/*
+			Release : v1.18
+			Testname: Kubectl, diff Deployment
+			Description: Create a Pod and a container with a given image. Configure Deployment to run 2 replicas. The number of running instances of the Pod MUST equal the number of replicas in the Deployment which is 2. Update the Deployment replicas to 3. Number of running instances of the Pod MUST be 3. Diff the original and live Deployments. Diff MUST be between number of replicas specified, 2, and number of replicas running, 3.
+		*/
+		ginkgo.It("should find diff with the replicas", func() {
+			deployment1Yaml := commonutils.SubstituteImageName(string(readTestFileOrDie(httpdDeployment1Filename)))
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+
+			ginkgo.By("deployment replicas number is 2")
```

Comment:
We need a test for pod. Somehow this is where most people set-up webhooks that may fail this process.

Reply:
Ah okay, will add a test for dry-run on a Pod